collections/_release_4_5/entry-general-editor-duplicate-projects-straight-from-the-project-manager.md

```diff
@@ -0,0 +1,17 @@
+---
+type: entry
+section: general
+subsection: editor
+rank: 8
+importance: 4
+anchor: duplicate-projects-straight-from-the-project-manager
+title: Duplicate projects straight from the Project manager
+text: |
+  There’s now a dedicated "Duplicate" button in the Project manager, making it easy to backup an existing project, or to fork a new one.
+
+  We also figured that it would be a great tool to prevent losses if you update the engine version during development. If we detect that irreversible changes would occur due to the update, we’ll ask you if you want to back up your project first.
```

Comment:
```suggestion
  We also figured that it would be a great tool to prevent losses if you update the engine version during development. If we detect that irreversible changes would occur due to the update, we’ll ask if you want to back up your project first.
```
Redundant and slows down the flow IMO